athlinks-schema.ttl

```diff
@@ -66,13 +72,22 @@ athlinks:AthleticRace
   rdfs:comment "An athletic race, a distinct course." ;
   rdfs:label "Athletic Race" ;
   rdfs:hasProperty athlinks:raceType ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:giveaway  ;
+  rdfs:hasProperty athlinks:require_phone ;
```

Comment:
We aren't using the rdfs:hasProperty notation anymore. simply need to remove these lines.